examples/sample_01/main.tf

```diff
@@ -0,0 +1,92 @@
+provider "google" {
+  credentials = file(var.credentials_file)
```

Comment:
do not specify the credentials here, use the application default. 
https://cloud.google.com/sdk/gcloud/reference/auth/application-default/login
It is is much safer this way